reconcile/status.py

```diff
@@ -2,3 +2,4 @@ class State:
     SUCCESS = 0
     ERROR = 1
     DATA_CHANGED = 3
+    SIGKILL = -9
```

Comment:
To be honest I don't understand why we are seeing -9 instead of 137. Everywhere I look I see that the PID cause by a signal is 128 + signal code.

Reply:
documented here: https://docs.python.org/3/library/subprocess.html#subprocess.Popen.poll